bindings/java/src/main/native/Makefile.am

```diff
@@ -0,0 +1,48 @@
+JDK = $(JAVA_HOME)
+topdir=../../../../..
+javadir=$(topdir)/bindings/java
+MVN=@cd $(javadir) && mvn
+ucm_lib=ucm/.libs
+ucs_lib=ucs/.libs
+uct_lib=uct/.libs
+ucp_lib=ucp/.libs
+sources=bridge.cc request_util.cc
+
+# additional include paths necessary to compile the C library
+AM_CPPFLAGS = -fPIC -DPIC -Werror \
+		-I$(JDK)/include -I$(JDK)/include/linux \
+		-I$(topdir)/src/
+
+AM_CXXFLAGS = -std=c++11
+
+
+lib_LTLIBRARIES = libjucx.la
+
+libjucx_la_SOURCES = $(sources)
+
+libjucx_la_LDFLAGS = -shared -rdynamic -Wl,-rpath,${prefix}/lib
+
+libjucx_la_LIBADD = -L$(topdir)/src/$(ucm_lib) -L$(topdir)/src/$(ucs_lib) -L$(topdir)/src/$(uct_lib) -L$(topdir)/src/$(ucp_lib) -lucm -lucs -luct -lucp
```

Comment:
long line, pls split with \
also, can we link with the ".la" files, like gtest for example, instead of passing "-L" directives?